src/elm/View/Form/Select.elm

```diff
@@ -1,15 +1,18 @@
-module View.Form.Select exposing (init, toHtml, withOption)
+{- module View.Form.Select exposing (init, toHtml, withOption)
```

Comment:
Same here. Maybe it would be best to revert the merge and do it again but if a conflict shows up on these files, just take `theirs` and it should be fine.

Reply:
I was having couple of errors when attempting to push the commit, elm-review was failing because of unused modules (View.Form.Select and View.Form.Input), as I was trying to fix it elm created these changes on save. I thought I had revert it.

Reply:
Alright. What I do in those cases is skip commit hooks using the `--no-verify` flag. Just make sure the review passes on your last commit before asking for a review :)

Reply:
Thanks! I will take a look at that!